City-of-Helsinki/drupal-hdbt-admin
# UHF-4929: Remove misleading help text

# Block library fixes [UHF-4929](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4929)
Remove misleading help text from paragraph creation page.

## What was done
* Removed the text form template file
* Removing the text made the whole template file obsolete, deleted the file.

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT Admin theme
    * `composer require drupal/hdbt_admin:dev-UHF-4929_block_library`
* Run `make drush-cr`

## How to test
* Go to /admin/content/paragraphs/add/default

What to expect:
There should be no help text between the title (`<h1>`) and the first input field.
